code/modules/overmap/ships/landable.dm

```diff
@@ -116,12 +116,14 @@
 		landmark.shuttle_name = shuttle
 		LAZYDISTINCTADD(initial_generic_waypoints, landmark.landmark_tag) // this is us being user-friendly: it means we register it properly regardless of whether the mapper put the tag in initial_restricted_waypoints
 
+	/* //#OUTREACH: Since we delete those right when creating them, might as well just comment this out
```

Comment:
This is fine, but you might as well delete the part of the code that actually deletes them as well.

Reply:
Just for reference, that was to fix the hard deletes. 
I'd suggest maybe doing that in another PR? Because this one is extremely bloated as it is.